integreat_cms/cms/templates/pages/page_tree.html

```diff
@@ -11,7 +11,7 @@
 {% with filter_form.filters_visible as filters_visible %}
 <div class="table-header">
     <div class="flex flex-wrap justify-between gap-4">
-        <h1 class="heading">{% trans 'Page Tree' %}</h1>
+        <h1 class="heading">{% trans 'Page Tree' %} <button data-show-tutorial="page_tree_tutorial"><i data-feather="help-circle"></i></button></h1>
```

Comment:
```suggestion
        <h1 class="heading">{% trans 'Page Tree' %} <button data-show-tutorial="page_tree_tutorial" class="hover:text-blue-500"><i data-feather="help-circle" class="align-baseline"></i></button></h1>
```
I think this looks a bit better